README.md

```diff
@@ -16,25 +16,15 @@ U.S. Data Federation website
 
 ### Setup
 
-Install the dependencies.
-
-    $ bundle install
-
-Build the Jekyll site.
-
-    $ bundle exec jekyll build
-
-Serve the site locally.
-
-    $ bundle exec jekyll serve
-
-Open your web browser to [localhost:4000](http://localhost:4000/).
-
-### Tests
-
-Check for broken links
-
-    $ bundle exec htmlproofer --check-html _site
+After you've confirmed the above requirements, all lifecycle operations can be run with Make commands (ex `make install`).
+
+| Make Command | Alias for                                    | Description                                                |
+| ------------ | -------------------------------------------- | ---------------------------------------------------------- |
+| install      | `bundle install`                             | Install dependencies (pinned in Gemfile.lock, if it exists |
+| update       | `bundle update`                              | Update installed dependencies; Reacreate Gemfile.lock      |
+| build        | `bundle exec jekyll build`                   | Build the site                                             |
+| serve        | `bundle exec jekyll serve`                   | Serve a local build                                        |
+| testlinks    | `bundle exec htmlproofer --check-html _site` | Check for broken links                                     |
```

Comment:
Is the alias important?  I feel like it's too much detail for the README

Reply:
i'm fine with taking them out.